.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,47 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
```

Comment:
You should add a filter to trigger the workflow on Go files
```
    paths:
      - 'go.mod'
      - '**.go'
      - '.github/workflows/golangci-lint.yaml'
```

Reply:
I am using the --new-from-rev=origin/main option in the golangci-lint workflow, which ensures that only changes introduced in the branch since origin/main are linted. This approach inherently limits linting to modified files, including Go files, without requiring an additional paths filter.

Reply:
Dans l'action : `only-new-issues: true`